tests/Rules/NarrowPublicClassMethodParamTypeRule/Fixture/ExplicitlyNullableParams.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\TypePerfect\Tests\Rules\NarrowPublicClassMethodParamTypeRule\Fixture;
+
+final class ExplicitlyNullableParams
+{
+    public function test(
+        ?bool $bool = true,
+    ): bool
+    {
+        return $bool;
+    }
+
+    static public function run(): void
+    {
+        $skipDateTimeMix = new ExplicitlyNullableParams();
+        $skipDateTimeMix->test(false);
```

Comment:
This is actually expected, as only `bool` value is passed. Never `null`

If `null` is passed as well, there is no error :+1: